pkg/operator/ceph/file/controller.go

```diff
@@ -178,7 +178,12 @@ func (r *ReconcileCephFilesystem) reconcile(request reconcile.Request) (reconcil
 			return reconcile.Result{}, nil
 		}
 
-		logger.Debugf("CephCluster resource not ready in namespace %q, retrying in %q.", request.NamespacedName.Namespace, opcontroller.WaitForRequeueIfCephClusterNotReadyAfter.String())
+		if reconcileResponse.RequeueAfter.String() == "0s" {
+			logger.Debugf("CephCluster resource does not exist in namespace %q, exponential backoff retry.", request.NamespacedName.Namespace)
```

Comment:
- Why do we need to log if it's exponential or not? Seems like we'll see debug logging for other actions being taken and we can see the timestamps for the retry anyway.
- If we do keep the debug logging, how about a helper to avoid the duplication in several places?

Reply:
> * Why do we need to log if it's exponential or not? Seems like we'll see debug logging for other actions being taken and we can see the timestamps for the retry anyway.

I thought you suggested to log the retry info at some point, that's why I came up with this. But we can make it more generic.

> * If we do keep the debug logging, how about a helper to avoid the duplication in several places?

The problem with that is that the helper package name is generic, so you actually don't know which reconcile is waiting... That's why I log in every controller...